Modules/Core/ImageFunction/include/itkRayCastInterpolateImageFunction.hxx

```diff
@@ -1003,29 +1003,29 @@ RayCastInterpolateImageFunction<TInputImage, TCoordinate>::RayCastHelper::Reset(
 
   else
   {
-    for (int i = 0; i < 3; ++i)
+    for (double & i : m_RayVoxelStartPosition)
     {
-      m_RayVoxelStartPosition[i] = 0.;
+      i = 0.;
```

Comment:
For these and similar, `i` has a different mean than the typical index integer counter, so it could be called `it`, `iter`,`position`, etc.

Reply:
It was easier and quicker than I anticipated.  Done!